src/NRediSearch/Client.cs

```diff
@@ -611,10 +790,12 @@ public AggregationResult Aggregate(AggregationRequest query)
 
             return new AggregationResult(resp);
         }
+
         /// <summary>
         /// Perform an aggregate query
         /// </summary>
         /// <param name="query">The query to watch</param>
+        [Obsolete("Use `AggregateAsync` method that takes an `AggregationBuilder`.")]
```

Comment:
why? I'm fuzzy on what the difference is between these APIs, since they look *very* similar; could do with some discussion here, I think

Reply:
I hate to "be that guy", but I have no idea. Maybe @gkorland could shed some light on this?

Reply:
To be clear, my intent was to bring parity between the two libraries. 😬

Reply:
no need to add AggregationRequest